tox.ini

```diff
@@ -121,43 +108,27 @@ commands=
     coverage run --source='.' manage.py test {posargs}
 
 
-[fast]
-# Configuration values necessary to run unittests without migrations.
-# Note: This is not an env will not run if invoked. Use an invocation of:
-#
-#   tox -e unittest-py{36}-dj{111}-wag{23}-fast
-#
-# To run unit tests.
-setenv=
-    DJANGO_SETTINGS_MODULE=cfgov.settings.test_nomigrations
-
-
-[slow]
-# Configuration values necessary to run unittests with migrations.
-# Note: This is not an env will not run if invoked. Use an invocation of:
-#
-#   tox -e unittest-py{36}-dj{111}-wag{23}-slow
-#
-# To run unit tests.
-setenv=
-    DJANGO_SETTINGS_MODULE=cfgov.settings.test
-
+[current]
+# Configuration necessary for the production versions Python, Django and 
+# Wagtail
+# Note: This is not an env will not run if invoked.
+basepython=python3.6
+deps=
+    -r{toxinidir}/requirements/django.txt
+    -r{toxinidir}/requirements/wagtail.txt
 
-[missing-migrations]
-# Extra configuration values used as part of slow environments.
-#
-# When running on Travis, we want to verify that Django migrations are
-# up-to-date. But we don't want to do that when running tox locally. We
-# accomplish this by having Travis set the TRAVIS_CHECK_MIGRATIONS
-# environment variable with the check migrations command.
-#
-# When not running on Travis, we want this to be a noop.
-commands=
-    {env:TRAVIS_CHECK_MIGRATIONS:echo "Skipping Travis-only step"}
+[future]
+# Configuration necessary for upcoming versions of Python, Django, and 
+# Wagtail
+# Note: This is not an env will not run if invoked.
+basepython=python3.6
+deps=
+    Django>=2.2,<2.3
+    wagtail>=2.3,<2.4
```

Comment:
Is it worth noting here that this section will need to be manually changed when we move the chains (unlike the 'current' configs)?

Reply:
Yes, it probably is.

Reply:
I've taken a stab at this in 3f28e5e. What do you think?